rubygems-3.6.2/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,132 @@
+# RubyGems and Bundler Code of Conduct
```

Comment:
we need to keep pulling the changes from the Master otherwise we overwrite the changes from other PRs that were pushed into the Master before us